Description: Fix the manpage-has-bad-whatis-entry warning
 The manpage format specifies that the NAME section should have a short name
 and a description separated by "\-". In the source document for each manpage,
 two hyphens are used ("--") and the rendered is converting that to "\[en]",
 which is incorrect and triggers this lintian warning:
 https://lintian.debian.org/tags/manpage-has-bad-whatis-entry.html
Author: Andreas Hasenack <andreas@canonical.com>
Bug: https://github.com/pmem/issues/issues/914
Last-Update: 2018-07-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/doc/generated/libpmem.7 b/doc/generated/libpmem.7
index 40727c5..b6a2a65 100644
--- a/doc/generated/libpmem.7
+++ b/doc/generated/libpmem.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libpmem\f[] \[en] persistent memory support library
+\f[B]libpmem\f[] \- persistent memory support library
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/libpmemblk.7 b/doc/generated/libpmemblk.7
index c04afd6..636e177 100644
--- a/doc/generated/libpmemblk.7
+++ b/doc/generated/libpmemblk.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libpmemblk\f[] \[en] persistent memory resident array of blocks
+\f[B]libpmemblk\f[] \- persistent memory resident array of blocks
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/libpmemcto.7 b/doc/generated/libpmemcto.7
index a60d7ea..c36a77e 100644
--- a/doc/generated/libpmemcto.7
+++ b/doc/generated/libpmemcto.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libpmemcto\f[] \[en] close\-to\-open persistence (EXPERIMENTAL)
+\f[B]libpmemcto\f[] \- close\-to\-open persistence (EXPERIMENTAL)
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/libpmemlog.7 b/doc/generated/libpmemlog.7
index 2059be5..67742f4 100644
--- a/doc/generated/libpmemlog.7
+++ b/doc/generated/libpmemlog.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libpmemlog\f[] \[en] persistent memory resident log file
+\f[B]libpmemlog\f[] \- persistent memory resident log file
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/libpmemobj.7 b/doc/generated/libpmemobj.7
index 445e388..a3c02ab 100644
--- a/doc/generated/libpmemobj.7
+++ b/doc/generated/libpmemobj.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libpmemobj\f[] \[en] persistent memory transactional object store
+\f[B]libpmemobj\f[] \- persistent memory transactional object store
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/libpmempool.7 b/doc/generated/libpmempool.7
index cf1c1b9..1327132 100644
--- a/doc/generated/libpmempool.7
+++ b/doc/generated/libpmempool.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libpmempool\f[] \[en] persistent memory pool management library
+\f[B]libpmempool\f[] \- persistent memory pool management library
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/librpmem.7 b/doc/generated/librpmem.7
index 1675ad0..b32c3ee 100644
--- a/doc/generated/librpmem.7
+++ b/doc/generated/librpmem.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]librpmem\f[] \[en] remote persistent memory support library
+\f[B]librpmem\f[] \- remote persistent memory support library
 (EXPERIMENTAL)
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/libvmem.7 b/doc/generated/libvmem.7
index 5d3a5a3..6dbf770 100644
--- a/doc/generated/libvmem.7
+++ b/doc/generated/libvmem.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libvmem\f[] \[en] volatile memory allocation library
+\f[B]libvmem\f[] \- volatile memory allocation library
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/libvmmalloc.7 b/doc/generated/libvmmalloc.7
index 9a80e68..8488bfd 100644
--- a/doc/generated/libvmmalloc.7
+++ b/doc/generated/libvmmalloc.7
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]libvmmalloc\f[] \[en] general purpose volatile memory allocation
+\f[B]libvmmalloc\f[] \- general purpose volatile memory allocation
 library
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/oid_is_null.3 b/doc/generated/oid_is_null.3
index da9aff4..b7b0f31 100644
--- a/doc/generated/oid_is_null.3
+++ b/doc/generated/oid_is_null.3
@@ -36,7 +36,7 @@
 \f[B]OID_IS_NULL\f[](), \f[B]OID_EQUALS\f[](),
 \f[B]pmemobj_direct\f[](), \f[B]pmemobj_oid\f[](),
 \f[B]pmemobj_type_num\f[](), \f[B]pmemobj_pool_by_oid\f[](),
-\f[B]pmemobj_pool_by_ptr\f[]() \[en] functions that allow mapping
+\f[B]pmemobj_pool_by_ptr\f[]() \- functions that allow mapping
 operations between object addresses, object handles, oids or type
 numbers
 .SH SYNOPSIS
diff --git a/doc/generated/pmem_flush.3 b/doc/generated/pmem_flush.3
index a31e23f..5e8a85a 100644
--- a/doc/generated/pmem_flush.3
+++ b/doc/generated/pmem_flush.3
@@ -36,7 +36,7 @@
 \f[B]pmem_flush\f[](), \f[B]pmem_drain\f[](), \f[B]pmem_persist\f[](),
 \f[B]pmem_msync\f[](), \f[B]pmem_deep_flush\f[](),
 \f[B]pmem_deep_drain\f[](), \f[B]pmem_deep_persist\f[](),
-\f[B]pmem_has_hw_drain\f[](), \f[B]pmem_has_auto_flush\f[]() \[en] check
+\f[B]pmem_has_hw_drain\f[](), \f[B]pmem_has_auto_flush\f[]() \- check
 persistency, store persistent data and delete mappings
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmem_is_pmem.3 b/doc/generated/pmem_is_pmem.3
index 5471024..d9a8d35 100644
--- a/doc/generated/pmem_is_pmem.3
+++ b/doc/generated/pmem_is_pmem.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmem_is_pmem\f[](), \f[B]pmem_map_file\f[](), \f[B]pmem_unmap\f[]()
-\[en] check persistency, create and delete mappings
+\- check persistency, create and delete mappings
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmem_memmove_persist.3 b/doc/generated/pmem_memmove_persist.3
index 728d2c9..1b6cdd9 100644
--- a/doc/generated/pmem_memmove_persist.3
+++ b/doc/generated/pmem_memmove_persist.3
@@ -35,7 +35,7 @@
 .PP
 \f[B]pmem_memmove_persist\f[](), \f[B]pmem_memcpy_persist\f[](),
 \f[B]pmem_memset_persist\f[](), \f[B]pmem_memmove_nodrain\f[](),
-\f[B]pmem_memcpy_nodrain\f[](), \f[B]pmem_memset_nodrain\f[]() \[en]
+\f[B]pmem_memcpy_nodrain\f[](), \f[B]pmem_memset_nodrain\f[]() \-
 functions that provide optimized copying to persistent memory
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemblk_bsize.3 b/doc/generated/pmemblk_bsize.3
index 61220e4..426fd32 100644
--- a/doc/generated/pmemblk_bsize.3
+++ b/doc/generated/pmemblk_bsize.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmemblk_bsize\f[](), \f[B]pmemblk_nblock\f[]() \[en] check number
+\f[B]pmemblk_bsize\f[](), \f[B]pmemblk_nblock\f[]() \- check number
 of available blocks or usable space in block memory pool
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemblk_create.3 b/doc/generated/pmemblk_create.3
index fa87b75..a4d166c 100644
--- a/doc/generated/pmemblk_create.3
+++ b/doc/generated/pmemblk_create.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemblk_create\f[](), \f[B]pmemblk_open\f[](),
-\f[B]pmemblk_close\f[](), \f[B]pmemblk_check\f[]() \[en] create, open,
+\f[B]pmemblk_close\f[](), \f[B]pmemblk_check\f[]() \- create, open,
 close and validate block pool
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemblk_read.3 b/doc/generated/pmemblk_read.3
index 0a58aad..ce12ebf 100644
--- a/doc/generated/pmemblk_read.3
+++ b/doc/generated/pmemblk_read.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmemblk_read\f[](), \f[B]pmemblk_write\f[]() \[en] read or write a
+\f[B]pmemblk_read\f[](), \f[B]pmemblk_write\f[]() \- read or write a
 block from a block memory pool
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemblk_set_zero.3 b/doc/generated/pmemblk_set_zero.3
index ec62e0d..3ea8eb7 100644
--- a/doc/generated/pmemblk_set_zero.3
+++ b/doc/generated/pmemblk_set_zero.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmemblk_set_zero\f[](), \f[B]pmemblk_set_error\f[]() \[en] block
+\f[B]pmemblk_set_zero\f[](), \f[B]pmemblk_set_error\f[]() \- block
 management functions
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemcto_aligned_alloc.3 b/doc/generated/pmemcto_aligned_alloc.3
index db45b1f..4483552 100644
--- a/doc/generated/pmemcto_aligned_alloc.3
+++ b/doc/generated/pmemcto_aligned_alloc.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_aligned_alloc \[en] allocate aligned memory
+pmemcto_aligned_alloc \- allocate aligned memory
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemcto_malloc.3 b/doc/generated/pmemcto_malloc.3
index bbe26cd..2fe6ff6 100644
--- a/doc/generated/pmemcto_malloc.3
+++ b/doc/generated/pmemcto_malloc.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_malloc, pmemcto_free, pmemcto_calloc, pmemcto_realloc \[en]
+pmemcto_malloc, pmemcto_free, pmemcto_calloc, pmemcto_realloc \-
 allocate and free persistent memory
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemcto_malloc_usable_size.3 b/doc/generated/pmemcto_malloc_usable_size.3
index e17b0ea..edd5f6f 100644
--- a/doc/generated/pmemcto_malloc_usable_size.3
+++ b/doc/generated/pmemcto_malloc_usable_size.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_malloc_usable_size \[en] obtain size of block of memory
+pmemcto_malloc_usable_size \- obtain size of block of memory
 allocated from pool
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemcto_open.3 b/doc/generated/pmemcto_open.3
index f41f230..5912608 100644
--- a/doc/generated/pmemcto_open.3
+++ b/doc/generated/pmemcto_open.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemcto_create\f[](), \f[B]pmemcto_open\f[](),
-\f[B]pmemcto_close\f[](), \f[B]pmemcto_check\f[]() \[en] create, open,
+\f[B]pmemcto_close\f[](), \f[B]pmemcto_check\f[]() \- create, open,
 close and validate close\-to\-open persistence pool
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemcto_set_root_pointer.3 b/doc/generated/pmemcto_set_root_pointer.3
index 451e6b6..9402b2a 100644
--- a/doc/generated/pmemcto_set_root_pointer.3
+++ b/doc/generated/pmemcto_set_root_pointer.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_set_root_pointer, pmemcto_get_root_pointer \[en] set or obtain
+pmemcto_set_root_pointer, pmemcto_get_root_pointer \- set or obtain
 the root object pointer
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemcto_stats_print.3 b/doc/generated/pmemcto_stats_print.3
index 4cfec09..af28bc8 100644
--- a/doc/generated/pmemcto_stats_print.3
+++ b/doc/generated/pmemcto_stats_print.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_stats_print \[en] write human\-readable memory pool statistics
+pmemcto_stats_print \- write human\-readable memory pool statistics
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemcto_strdup.3 b/doc/generated/pmemcto_strdup.3
index ba13cd2..d080afe 100644
--- a/doc/generated/pmemcto_strdup.3
+++ b/doc/generated/pmemcto_strdup.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_strdup \[en] duplicate a string
+pmemcto_strdup \- duplicate a string
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemcto_wcsdup.3 b/doc/generated/pmemcto_wcsdup.3
index f867aba..2045cac 100644
--- a/doc/generated/pmemcto_wcsdup.3
+++ b/doc/generated/pmemcto_wcsdup.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-pmemcto_wcsdup \[en] duplicate a wide\-char string
+pmemcto_wcsdup \- duplicate a wide\-char string
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemlog_append.3 b/doc/generated/pmemlog_append.3
index 1a5bfef..787f470 100644
--- a/doc/generated/pmemlog_append.3
+++ b/doc/generated/pmemlog_append.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmemlog_append\f[](), \f[B]pmemlog_appendv\f[]() \[en] append bytes
+\f[B]pmemlog_append\f[](), \f[B]pmemlog_appendv\f[]() \- append bytes
 to the persistent memory resident log file
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemlog_create.3 b/doc/generated/pmemlog_create.3
index a313879..82c9b40 100644
--- a/doc/generated/pmemlog_create.3
+++ b/doc/generated/pmemlog_create.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemlog_create\f[](), \f[B]pmemlog_open\f[](),
-\f[B]pmemlog_close\f[](), \f[B]pmemlog_check\f[]() \[en] create, open,
+\f[B]pmemlog_close\f[](), \f[B]pmemlog_check\f[]() \- create, open,
 close and validate persistent memory resident log file
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemlog_nbyte.3 b/doc/generated/pmemlog_nbyte.3
index df92b2a..0ebed5a 100644
--- a/doc/generated/pmemlog_nbyte.3
+++ b/doc/generated/pmemlog_nbyte.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmemlog_nbyte\f[]() \[en] checks the amount of usable space in the
+\f[B]pmemlog_nbyte\f[]() \- checks the amount of usable space in the
 log pool.
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemlog_tell.3 b/doc/generated/pmemlog_tell.3
index 09bee21..2b19999 100644
--- a/doc/generated/pmemlog_tell.3
+++ b/doc/generated/pmemlog_tell.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemlog_tell\f[](), \f[B]pmemlog_rewind\f[](),
-\f[B]pmemlog_walk\f[]() \[en] checks current write point for the log or
+\f[B]pmemlog_walk\f[]() \- checks current write point for the log or
 walks through the log
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_action.3 b/doc/generated/pmemobj_action.3
index 1dcc82b..b050704 100644
--- a/doc/generated/pmemobj_action.3
+++ b/doc/generated/pmemobj_action.3
@@ -36,7 +36,7 @@
 \f[B]pmemobj_reserve\f[](), \f[B]pmemobj_xreserve\f[](),
 \f[B]pmemobj_set_value\f[](), \f[B]pmemobj_publish\f[](),
 \f[B]pmemobj_tx_publish\f[](), \f[B]pmemobj_cancel\f[](),
-\f[B]POBJ_RESERVE_NEW\f[](), \f[B]POBJ_RESERVE_ALLOC\f[]() \[en] Delayed
+\f[B]POBJ_RESERVE_NEW\f[](), \f[B]POBJ_RESERVE_ALLOC\f[]() \- Delayed
 atomicity actions (EXPERIMENTAL)
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_alloc.3 b/doc/generated/pmemobj_alloc.3
index ee84b56..db99343 100644
--- a/doc/generated/pmemobj_alloc.3
+++ b/doc/generated/pmemobj_alloc.3
@@ -39,7 +39,7 @@
 \f[B]pmemobj_wcsdup\f[](), \f[B]pmemobj_alloc_usable_size\f[](),
 \f[B]POBJ_NEW\f[](), \f[B]POBJ_ALLOC\f[](), \f[B]POBJ_ZNEW\f[](),
 \f[B]POBJ_ZALLOC\f[](), \f[B]POBJ_REALLOC\f[](),
-\f[B]POBJ_ZREALLOC\f[](), \f[B]POBJ_FREE\f[]() \[en] non\-transactional
+\f[B]POBJ_ZREALLOC\f[](), \f[B]POBJ_FREE\f[]() \- non\-transactional
 atomic allocations
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_ctl_get.3 b/doc/generated/pmemobj_ctl_get.3
index b780c62..e362c2d 100644
--- a/doc/generated/pmemobj_ctl_get.3
+++ b/doc/generated/pmemobj_ctl_get.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemobj_ctl_get\f[](), \f[B]pmemobj_ctl_set\f[](),
-\f[B]pmemobj_ctl_exec\f[]() \[en] Query and modify libpmemobj internal
+\f[B]pmemobj_ctl_exec\f[]() \- Query and modify libpmemobj internal
 behavior (EXPERIMENTAL)
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_first.3 b/doc/generated/pmemobj_first.3
index bfb9fc0..8933e55 100644
--- a/doc/generated/pmemobj_first.3
+++ b/doc/generated/pmemobj_first.3
@@ -37,7 +37,7 @@
 \f[B]POBJ_FIRST\f[](), \f[B]POBJ_FIRST_TYPE_NUM\f[](),
 \f[B]POBJ_NEXT\f[](), \f[B]POBJ_NEXT_TYPE_NUM\f[](),
 \f[B]POBJ_FOREACH\f[](), \f[B]POBJ_FOREACH_SAFE\f[](),
-\f[B]POBJ_FOREACH_TYPE\f[](), \f[B]POBJ_FOREACH_SAFE_TYPE\f[]() \[en]
+\f[B]POBJ_FOREACH_TYPE\f[](), \f[B]POBJ_FOREACH_SAFE_TYPE\f[]() \-
 pmemobj container operations
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_list_insert.3 b/doc/generated/pmemobj_list_insert.3
index 0f76464..d055379 100644
--- a/doc/generated/pmemobj_list_insert.3
+++ b/doc/generated/pmemobj_list_insert.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemobj_list_insert\f[](), \f[B]pmemobj_list_insert_new\f[](),
-\f[B]pmemobj_list_move\f[](), \f[B]pmemobj_list_remove\f[]() \[en]
+\f[B]pmemobj_list_move\f[](), \f[B]pmemobj_list_remove\f[]() \-
 non\-transactional persistent atomic lists functions
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_memcpy_persist.3 b/doc/generated/pmemobj_memcpy_persist.3
index 1d3e04f..c222c33 100644
--- a/doc/generated/pmemobj_memcpy_persist.3
+++ b/doc/generated/pmemobj_memcpy_persist.3
@@ -35,7 +35,7 @@
 .PP
 \f[B]pmemobj_memcpy_persist\f[](), \f[B]pmemobj_memset_persist\f[](),
 \f[B]pmemobj_persist\f[](), \f[B]pmemobj_flush\f[](),
-\f[B]pmemobj_drain\f[]() \[en] low\-level memory manipulation functions
+\f[B]pmemobj_drain\f[]() \- low\-level memory manipulation functions
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemobj_mutex_zero.3 b/doc/generated/pmemobj_mutex_zero.3
index 98a6c79..4eb80a7 100644
--- a/doc/generated/pmemobj_mutex_zero.3
+++ b/doc/generated/pmemobj_mutex_zero.3
@@ -45,7 +45,7 @@
 .PP
 \f[B]pmemobj_cond_zero\f[](), \f[B]pmemobj_cond_broadcast\f[](),
 \f[B]pmemobj_cond_signal\f[](), \f[B]pmemobj_cond_timedwait\f[](),
-\f[B]pmemobj_cond_wait\f[]() \[en] pmemobj synchronization primitives
+\f[B]pmemobj_cond_wait\f[]() \- pmemobj synchronization primitives
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemobj_open.3 b/doc/generated/pmemobj_open.3
index 2404f57..b115c34 100644
--- a/doc/generated/pmemobj_open.3
+++ b/doc/generated/pmemobj_open.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemobj_open\f[](), \f[B]pmemobj_create\f[](),
-\f[B]pmemobj_close\f[](), \f[B]pmemobj_check\f[]() \[en] create, open,
+\f[B]pmemobj_close\f[](), \f[B]pmemobj_check\f[]() \- create, open,
 close and validate persistent memory transactional object store
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_root.3 b/doc/generated/pmemobj_root.3
index 6f1a648..1a8fed8 100644
--- a/doc/generated/pmemobj_root.3
+++ b/doc/generated/pmemobj_root.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmemobj_root\f[](), \f[B]pmemobj_root_construct\f[]()
-\f[B]POBJ_ROOT\f[](), \f[B]pmemobj_root_size\f[]() \[en] root object
+\f[B]POBJ_ROOT\f[](), \f[B]pmemobj_root_size\f[]() \- root object
 management
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmemobj_tx_add_range.3 b/doc/generated/pmemobj_tx_add_range.3
index 92781be..0735733 100644
--- a/doc/generated/pmemobj_tx_add_range.3
+++ b/doc/generated/pmemobj_tx_add_range.3
@@ -44,7 +44,7 @@
 \f[B]TX_XADD_FIELD_DIRECT\f[](),
 .PP
 \f[B]TX_SET\f[](), \f[B]TX_SET_DIRECT\f[](), \f[B]TX_MEMCPY\f[](),
-\f[B]TX_MEMSET\f[]() \[en] transactional object manipulation
+\f[B]TX_MEMSET\f[]() \- transactional object manipulation
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemobj_tx_alloc.3 b/doc/generated/pmemobj_tx_alloc.3
index 5634710..27e46a1 100644
--- a/doc/generated/pmemobj_tx_alloc.3
+++ b/doc/generated/pmemobj_tx_alloc.3
@@ -41,7 +41,7 @@
 \f[B]TX_NEW\f[](), \f[B]TX_ALLOC\f[](), \f[B]TX_ZNEW\f[](),
 \f[B]TX_ZALLOC\f[](), \f[B]TX_XALLOC\f[](), \f[B]TX_REALLOC\f[](),
 \f[B]TX_ZREALLOC\f[](), \f[B]TX_STRDUP\f[](), \f[B]TX_WCSDUP\f[](),
-\f[B]TX_FREE\f[]() \[en] transactional object manipulation
+\f[B]TX_FREE\f[]() \- transactional object manipulation
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmemobj_tx_begin.3 b/doc/generated/pmemobj_tx_begin.3
index 44043a2..e2a1c74 100644
--- a/doc/generated/pmemobj_tx_begin.3
+++ b/doc/generated/pmemobj_tx_begin.3
@@ -42,7 +42,7 @@
 .PP
 \f[B]TX_BEGIN_PARAM\f[](), \f[B]TX_BEGIN_CB\f[](), \f[B]TX_BEGIN\f[](),
 \f[B]TX_ONABORT\f[], \f[B]TX_ONCOMMIT\f[], \f[B]TX_FINALLY\f[],
-\f[B]TX_END\f[] \[en] transactional object manipulation
+\f[B]TX_END\f[] \- transactional object manipulation
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool-check.1 b/doc/generated/pmempool-check.1
index 663d091..3a9b40f 100644
--- a/doc/generated/pmempool-check.1
+++ b/doc/generated/pmempool-check.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-check\f[] \[en] check and repair persistent memory pool
+\f[B]pmempool\-check\f[] \- check and repair persistent memory pool
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool-create.1 b/doc/generated/pmempool-create.1
index b8bae92..13497b7 100644
--- a/doc/generated/pmempool-create.1
+++ b/doc/generated/pmempool-create.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-create\f[] \[en] create a persistent memory pool
+\f[B]pmempool\-create\f[] \- create a persistent memory pool
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool-dump.1 b/doc/generated/pmempool-dump.1
index 3354866..0e9a4a0 100644
--- a/doc/generated/pmempool-dump.1
+++ b/doc/generated/pmempool-dump.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-dump\f[] \[en] dump user data from persistent memory pool
+\f[B]pmempool\-dump\f[] \- dump user data from persistent memory pool
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool-info.1 b/doc/generated/pmempool-info.1
index 449f386..4dd7697 100644
--- a/doc/generated/pmempool-info.1
+++ b/doc/generated/pmempool-info.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-info\f[] \[en] show information about persistent memory
+\f[B]pmempool\-info\f[] \- show information about persistent memory
 pool
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmempool-rm.1 b/doc/generated/pmempool-rm.1
index 79b3f6f..38b6639 100644
--- a/doc/generated/pmempool-rm.1
+++ b/doc/generated/pmempool-rm.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-rm\f[] \[en] remove a persistent memory pool
+\f[B]pmempool\-rm\f[] \- remove a persistent memory pool
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool-sync.1 b/doc/generated/pmempool-sync.1
index e6e2e78..7c6747f 100644
--- a/doc/generated/pmempool-sync.1
+++ b/doc/generated/pmempool-sync.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-sync\f[] \[en] Synchronize replicas or their parts within
+\f[B]pmempool\-sync\f[] \- Synchronize replicas or their parts within
 a pool set.
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmempool-transform.1 b/doc/generated/pmempool-transform.1
index 578c975..392c7ea 100644
--- a/doc/generated/pmempool-transform.1
+++ b/doc/generated/pmempool-transform.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\-transform\f[] \[en] Modify internal structure of a pool
+\f[B]pmempool\-transform\f[] \- Modify internal structure of a pool
 set.
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pmempool.1 b/doc/generated/pmempool.1
index 43ba637..5051273 100644
--- a/doc/generated/pmempool.1
+++ b/doc/generated/pmempool.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool\f[] \[en] Persistent Memory Pool Management Tool
+\f[B]pmempool\f[] \- Persistent Memory Pool Management Tool
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool_check_init.3 b/doc/generated/pmempool_check_init.3
index c3872f1..972a68b 100644
--- a/doc/generated/pmempool_check_init.3
+++ b/doc/generated/pmempool_check_init.3
@@ -34,7 +34,7 @@
 .SH NAME
 .PP
 \f[B]pmempool_check_init\f[](), \f[B]pmempool_check\f[](),
-\f[B]pmempool_check_end\f[]() \[en] checks pmempool health
+\f[B]pmempool_check_end\f[]() \- checks pmempool health
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool_rm.3 b/doc/generated/pmempool_rm.3
index c4e896d..3ed0da0 100644
--- a/doc/generated/pmempool_rm.3
+++ b/doc/generated/pmempool_rm.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool_rm\f[]() \[en] remove persistent memory pool
+\f[B]pmempool_rm\f[]() \- remove persistent memory pool
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/pmempool_sync.3 b/doc/generated/pmempool_sync.3
index ff5fb5a..e9ffa14 100644
--- a/doc/generated/pmempool_sync.3
+++ b/doc/generated/pmempool_sync.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]pmempool_sync\f[](), \f[B]pmempool_transform\f[]() \[en] pool set
+\f[B]pmempool_sync\f[](), \f[B]pmempool_transform\f[]() \- pool set
 synchronization and transformation
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pobj_layout_begin.3 b/doc/generated/pobj_layout_begin.3
index 50e095d..e43b799 100644
--- a/doc/generated/pobj_layout_begin.3
+++ b/doc/generated/pobj_layout_begin.3
@@ -35,7 +35,7 @@
 .PP
 \f[B]POBJ_LAYOUT_BEGIN\f[](), \f[B]POBJ_LAYOUT_TOID\f[](),
 \f[B]POBJ_LAYOUT_ROOT\f[](), \f[B]POBJ_LAYOUT_NAME\f[](),
-\f[B]POBJ_LAYOUT_END\f[](), \f[B]POBJ_LAYOUT_TYPES_NUM\f[]() \[en]
+\f[B]POBJ_LAYOUT_END\f[](), \f[B]POBJ_LAYOUT_TYPES_NUM\f[]() \-
 persistent memory transactional object store layout
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/pobj_list_head.3 b/doc/generated/pobj_list_head.3
index 26092d0..7598446 100644
--- a/doc/generated/pobj_list_head.3
+++ b/doc/generated/pobj_list_head.3
@@ -52,7 +52,7 @@
 \f[B]POBJ_LIST_MOVE_ELEMENT_HEAD\f[](),
 \f[B]POBJ_LIST_MOVE_ELEMENT_TAIL\f[](),
 \f[B]POBJ_LIST_MOVE_ELEMENT_AFTER\f[](),
-\f[B]POBJ_LIST_MOVE_ELEMENT_BEFORE\f[]() \[en] type\-safe
+\f[B]POBJ_LIST_MOVE_ELEMENT_BEFORE\f[]() \- type\-safe
 non\-transactional persistent atomic lists
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/poolset.5 b/doc/generated/poolset.5
index 0bc727c..5400c10 100644
--- a/doc/generated/poolset.5
+++ b/doc/generated/poolset.5
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-poolset \[en] persistent memory pool configuration file format
+poolset \- persistent memory pool configuration file format
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/rpmem_create.3 b/doc/generated/rpmem_create.3
index 0d78bb9..22b8f6e 100644
--- a/doc/generated/rpmem_create.3
+++ b/doc/generated/rpmem_create.3
@@ -35,7 +35,7 @@
 .PP
 \f[B]rpmem_create\f[](), \f[B]rpmem_open\f[](),
 \f[B]rpmem_set_attr\f[](), \f[B]rpmem_close\f[](),
-\f[B]rpmem_remove\f[]() \[en] most commonly used functions for remote
+\f[B]rpmem_remove\f[]() \- most commonly used functions for remote
 access to \f[I]persistent memory\f[]
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/rpmem_persist.3 b/doc/generated/rpmem_persist.3
index e6cf5cf..6e91c4e 100644
--- a/doc/generated/rpmem_persist.3
+++ b/doc/generated/rpmem_persist.3
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]rpmem_persist\f[](), \f[B]rpmem_read\f[](), \[en] functions to copy
+\f[B]rpmem_persist\f[](), \f[B]rpmem_read\f[](), \- functions to copy
 and read remote pools
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/rpmemd.1 b/doc/generated/rpmemd.1
index 6789b99..deb6935 100644
--- a/doc/generated/rpmemd.1
+++ b/doc/generated/rpmemd.1
@@ -33,7 +33,7 @@
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .SH NAME
 .PP
-\f[B]rpmemd\f[] \[en] librpmem target node process (EXPERIMENTAL)
+\f[B]rpmemd\f[] \- librpmem target node process (EXPERIMENTAL)
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/toid_declare.3 b/doc/generated/toid_declare.3
index 50af4bb..513ea4d 100644
--- a/doc/generated/toid_declare.3
+++ b/doc/generated/toid_declare.3
@@ -38,7 +38,7 @@
 \f[B]TOID_VALID\f[](), \f[B]OID_INSTANCEOF\f[](),
 \f[B]TOID_ASSIGN\f[](), \f[B]TOID_IS_NULL\f[](), \f[B]TOID_EQUALS\f[](),
 \f[B]TOID_TYPEOF\f[](), \f[B]TOID_OFFSETOF\f[](), \f[B]DIRECT_RW\f[](),
-\f[B]D_RW\f[](), \f[B]DIRECT_RO\f[](), \f[B]D_RO\f[]() \[en] libpmemobj
+\f[B]D_RW\f[](), \f[B]DIRECT_RO\f[](), \f[B]D_RO\f[]() \- libpmemobj
 type safety mechanism
 .SH SYNOPSIS
 .IP
diff --git a/doc/generated/vmem_create.3 b/doc/generated/vmem_create.3
index bf341cd..a862eeb 100644
--- a/doc/generated/vmem_create.3
+++ b/doc/generated/vmem_create.3
@@ -35,7 +35,7 @@
 .PP
 \f[B]vmem_create\f[](), \f[B]vmem_create_in_region\f[](),
 \f[B]vmem_delete\f[](), \f[B]vmem_check\f[](),
-\f[B]vmem_stats_print\f[]() \[en] volatile memory pool management
+\f[B]vmem_stats_print\f[]() \- volatile memory pool management
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/doc/generated/vmem_malloc.3 b/doc/generated/vmem_malloc.3
index 107e57f..a085043 100644
--- a/doc/generated/vmem_malloc.3
+++ b/doc/generated/vmem_malloc.3
@@ -36,7 +36,7 @@
 \f[B]vmem_malloc\f[](), \f[B]vmem_calloc\f[](), \f[B]vmem_realloc\f[](),
 \f[B]vmem_free\f[](), \f[B]vmem_aligned_alloc\f[](),
 \f[B]vmem_strdup\f[](), \f[B]vmem_wcsdup\f[](),
-\f[B]vmem_malloc_usable_size\f[]() \[en] memory allocation related
+\f[B]vmem_malloc_usable_size\f[]() \- memory allocation related
 functions
 .SH SYNOPSIS
 .IP
